app/helpers/statuses_helper.rb

```diff
@@ -57,6 +57,19 @@ def status_description(status)
     components.compact_blank.join("\n\n")
   end
 
+  def preview_card_aspect_ratio_classname(preview_card)
```

Comment:
This is a reimplementation of the logic from the frontend for preview cards: https://github.com/mastodon/mastodon/blob/425311e1d95c8a64ddac6c724fca247b8b893a82/app/javascript/mastodon/features/status/components/card.jsx#L160